app/components/MinimalHeader.tsx

```diff
@@ -16,16 +19,23 @@ const getAssets = () => {
 
 const MinimalHeader = ({ className }: MinimalHeaderProps): JSX.Element => {
   const assets = useMemo(() => getAssets(), []);
+  const [showSidebar, setShowSidebar] = React.useState(false);
+  const { verificationState } = useOneClickVerification();
 
   return (
-    <div className={`flex h-16 ${className}`}>
-      <div className="flex items-center">
-        <img className="" src={assets.gitcoinLogo} alt="Gitcoin Logo" />
-        <img className="mx-3 md:mx-6" src={assets.logoLine} alt="Logo Line" />
-        <img className="h-8" src={assets.passportLogo} alt="Passport Logo" />
-        <div className="ml-3 text-2xl text-color-1">Passport</div>
+    <>
+      <OnchainSidebar isOpen={showSidebar} onClose={() => setShowSidebar(false)} />
+      <div className={`flex justify-between h-16 ${className}`}>
+        <div className="flex items-center">
+          <img className="" src={assets.gitcoinLogo} alt="Gitcoin Logo" />
+          <img className="mx-3 md:mx-6" src={assets.logoLine} alt="Logo Line" />
+          <img className="h-8" src={assets.passportLogo} alt="Passport Logo" />
+          <div className="ml-3 text-2xl text-color-1">Passport</div>
+        </div>
+        {verificationState.error !== undefined ||
+          (verificationState.success && <Notifications setShowSidebar={() => setShowSidebar(true)} />)}
```

Comment:
Also, I would make the notification widget more resilient.
For example if the oneClickVerification fails, I think we should still display the notification bell icon + the notifications we are able to determine in this case.

So I would check the `loading` is false (has finished - maybe we will need more than a boolean here), and regardless of the values of success and error I would display the notifications.

Reply:
This is the case currently. If `verificationState.error` is defined the Notifications component will be rendered https://github.com/gitcoinco/passport/blob/eff59719aff497530285ceac4887b74433dbfd5b/app/hooks/useOneClickVerification.tsx#L100. So if an error is thrown during the one click flow the notifications will still be rendered. 

I didn't use the loading key since it starts off as false. There was one case where if a user does not have any stamps, the one click flow exits. I added a success value there so notifications will be rendered.